.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,9 +4,6 @@ GitHub Issue: #
 <!-- Please check one or more that apply to this PR. -->
```

Comment:
You should adjust this text to explain that this repository is the canary and therefore all PRs should be about fixing the canary and not be about adding new features.

Reply:
updated